src/components/SummaryBar/SummaryBar.jsx

```diff
@@ -234,7 +235,7 @@ const SummaryBar = (props) => {
               <div className="border-red col-4 bg--white-smoke no-gutters" align="center">
                 <div className="py-1"> </div>
 
-                {matchUser || isAdmin ? (
+                {matchUser  ? (
```

Comment:
Based on the line 33 change `isAdmin`-> `role`, does line 237 need do `isAdmin`->`hasPermission(role, 'toggleSubmitForm')` and add `toggleSubmitForm` into `permissions.js`?

Reply:
Great suggestion! Just added it